python/pylibcudf/pylibcudf/tests/test_reshape.py

```diff
@@ -5,6 +5,17 @@
 from utils import assert_column_eq, assert_table_eq
 
 import pylibcudf as plc
+from pylibcudf.types import TypeId
+
+
+@pytest.fixture(scope="module")
+def np():
+    return pytest.importorskip("cupy")
+
+
+@pytest.fixture(scope="module")
+def cp():
+    return pytest.importorskip("cupy")
```

Comment:
Did you mean to import `cupy` as `np` here? Also I would prefer to use this at the module root, or in `conftest.py`, or perhaps with a separate test file, or simply inline this into the test(s) where it is used. Anything but a module-scope import-or-skip fixture would be better imo.
```suggestion
np = pytest.importorskip("numpy")
cp = pytest.importorskip("cupy")
```

Reply:
I just imported cupy in the test